dev/io.openliberty.microprofile.health.4.0.internal/src/io/openliberty/microprofile/health40/internal/HealthFileUtils.java

```diff
@@ -102,21 +102,13 @@ public static boolean isValidSystem() throws IOException {
         //Health Dir does not exist -> create and test write
         if (!healthDirFile.exists()) {
             if (!HealthFileUtils.createDirectory(healthDirFile)) {
-                //TODO: Create warning message with ID
-                if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
-                    Tr.debug(tc, "Unable to create the /health directory. The mpHealth file-based health check reporting will be disabled.");
-                }
-
+                Tr.warning(tc, "file.healthcheck.health.directory.create.fail.CWMMH0100W", healthDirFile.getAbsolutePath());
                 return false;
             }
 
             //Testing write.
             if (!HealthFileUtils.canWriteToDirectory(healthDirFile)) {
-                //TODO: Create warning message with ID
-                if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
-                    Tr.debug(tc, "Unable to write to the /health directory. The mpHealth file-based health check reporting will be disabled.");
-                }
-                return false;
+                Tr.warning(tc, "file.healthcheck.health.directory.create.fail.CWMMH0100W", healthDirFile.getAbsolutePath());
```

Comment:
Should this warning be `file.healthcheck.health.directory.write.fail.CWMMH0101W` instead?